activemq-broker/src/main/java/org/apache/activemq/broker/region/policy/DeadLetterStrategy.java

```diff
@@ -60,6 +71,16 @@ public interface DeadLetterStrategy {
      */
     public void setProcessNonPersistent(boolean processNonPersistent);
 
+    /**
+     * @return the PreserveDeliveryMode
```

Comment:
The comment needs to be updated.

Reply:
> @return the PreserveDeliveryMode

This is a new method which does return `PreserveDeliveryMode`. What exactly would you like to update?

Reply:
Yeah, I noticed the "is" style is used for boolean methods in a number of places so I chose to be consistent.

Reply:
yeah, is for boolean is the convetion